### ISSUE TITLE
Unable to parse input with a square bracket

### ISSUE THREAD

**Issue by ArsalanKhairani** [jcubic/jquery.terminal#389]
### Expected behavior
Input should parse successfully.

### Actual behavior
```
[USER]: Unexpected token d in JSON at position 3
SyntaxError: Unexpected token d in JSON at position 3
    at JSON.parse (<anonymous>)
    at parse_string (https://rawgit.com/jcubic/jquery.terminal/master/js/jquery.terminal.js:3508:29)
    at Object.parse_argument (https://rawgit.com/jcubic/jquery.terminal/master/js/jquery.terminal.js:3519:28)
    at https://rawgit.com/jcubic/jquery.terminal/master/js/jquery.terminal.js:3564:35
    at https://rawgit.com/jcubic/jquery.terminal/master/js/jquery.terminal.js:2810:24
    at Function.map (https://code.jquery.com/jquery-3.1.1.min.js:2:3409)
    at process_command (https://rawgit.com/jcubic/jquery.terminal/master/js/jquery.terminal.js:2804:26)
    at Object.split_command (https://rawgit.com/jcubic/jquery.terminal/master/js/jquery.terminal.js:3563:20)
    at Object.commands (https://rawgit.com/jcubic/jquery.terminal/master/js/jquery.terminal.js:4907:47)
    at Object.ENTER (https://rawgit.com/jcubic/jquery.terminal/master/js/jquery.terminal.js:1214:29)
```

### Steps to reproduce
```
> {"demo": ["error"]}
```

Works okay if you remove the space before `[`.
```
> {"demo":["error"]}
```

### Browser and OS
Google Chrome Version 65.0.3325.181 (Official Build) (64-bit) | Linux.
Tried on codepen and it's reproducible there as well.

**Comment by jcubic** [comment]
Fixed in devel branch

**Comment by ArsalanKhairani** [comment]
@jcubic: Appreciate the quick response 👍

**Comment by jcubic** [comment]
released in version 1.15.0
